cmd/state/commands/root.go

```
@@ -83,12 +84,12 @@ func genesisFromFile(genesisPath string) *types.Genesis {
 }
 
 func getChainGenesisAndConfig() (genesis *types.Genesis, chainConfig *chain2.Config) {
-	if chain == "" {
-		genesis, chainConfig = chainspec.MainnetGenesisBlock(), chainspec.MainnetChainConfig
-	} else {
-		genesis, chainConfig = chainspec.GenesisBlockByChainName(chain), chainspec.ChainConfigByChainName(chain)
+	name := chain
+	if name == "" {
+		name = networkname.Mainnet
 	}
-	return genesis, chainConfig
+	spec, _ := chainspec.ChainSpecByName(name)
```

Error is ignored here.